code/modules/shuttle/special.dm

```diff
@@ -405,3 +405,29 @@
 
 /obj/effect/decal/hammerandsickle/shuttleRotate(rotation)
 	setDir(angle2dir(rotation+dir2angle(dir))) // No parentcall, rest of the rotate code breaks the pixel offset.
+
+// flying dutchman cannons.
+/obj/machinery/power/emitter/energycannon/cannon
+	name = "cannon"
+	desc = "It's just the entry model. With a bit more booty, you could have gotten the cannon used by that most fearsome scoundrel Renee Rottingham himself."
+	projectile_type = /obj/projectile/bullet/cannonball
+	icon = 'icons/obj/machines/magic_emitter.dmi'
+	icon_state = "wabbajack_statue"
+	icon_state_on = "wabbajack_statue_on"
+	projectile_sound = 'sound/effects/meteorimpact.ogg'
+	active = FALSE
+	allow_switch_interact = FALSE
```

Comment:
i assume ur gonna link these to buttons in mapping

Reply:
Actually, kryson made cannons in the past, I'm going to use his cannons. This should be GUTTED!